### PR TITLE
added kill command

### PR DESCRIPTION
**Pull request by Shiimoe**
added kill command with suicide capabilities (I hate you btw)

### CODE REVIEW

**Comment by Demonstrandum** [comment]
Change the title and I might merge it.

**Comment by Shiimoe** [comment]
stop being a lil bitch and merge it

File: lib/action.ts [hunk 1]
```diff
@@ -138,7 +138,25 @@ const ACTIONS: Record<ActionType, Actions> = {
 			"https://img2.gelbooru.com/images/94/83/94833376ed7271f08c75385204fbcd1f.gif",
 			"https://img.rule34.xxx//images/1149/9362d2e884efc6e49b665be3268d4f02d66674a3.gif?1148722",
 		]
+	},
+	kill: {
+		title: "You have died.",
+		message: "You feel the life slowly draining from your body as your draw in your last breaths.",
```
Comment (Demonstrandum) [inline lib/action.ts]: The way `message` acts is like a verb, and the two people involved are placed either side of this message.

e.g  Person1 `messaage` Person2!  (`transitiveness: true`)
or Person2 `message` from Person1 (`transitiveness: false`)

Yeah?

File: lib/action.ts [hunk 2]
```diff
@@ -138,7 +138,25 @@ const ACTIONS: Record<ActionType, Actions> = {
...
+		emoji: 'skull',
+		images: [
+			"https://img2.gelbooru.com/images/7c/9c/7c9cdd15e03df0a2ba06b9bb9aa98180.png",
+		] 
+	},
+	suicide: {
+		title: "You have killed yourself.",
+		message: "Good riddance.",
```
Comment (Demonstrandum) [inline lib/action.ts]: Same goes for this.

File: lib/action.ts [hunk 3]
```diff
@@ -138,7 +138,25 @@ const ACTIONS: Record<ActionType, Actions> = {
...
+	},
+	kill: {
+		title: "You have died.",
+		message: "You feel the life slowly draining from your body as your draw in your last breaths.",
+		emoji: 'skull',
+		images: [
+			"https://img2.gelbooru.com/images/7c/9c/7c9cdd15e03df0a2ba06b9bb9aa98180.png",
+		] 
+	},
+	suicide: {
```
Comment (Demonstrandum) [inline lib/action.ts]: Suicide doesn't really work for two people, you should make this its own command.
`Action` commands are interactions between two people.

File: lib/commands/kill.ts [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import { Message } from 'discord.js';
+import Action from '../action';
+
+export default (home_scope: HomeScope) => {
+	const { message, args }
+			: { message: Message, args: string[] } = home_scope;
+
+	if (args.length === 0 || message.mentions.users.size === 0)
+		return message.channel.send(Action.get('suicide', message);
```
Comment (Demonstrandum) [inline lib/commands/kill.ts]: You want to not use `Action`s for this message.

**Review by Demonstrandum — CHANGES_REQUESTED** [review]
You need to make some changes.

You also uploaded two swap files? (`.swp`)

Please delete them.

**Comment by Demonstrandum** [comment]
Checks did not pass:
```
##[error]lib/commands/kill.ts(9,61): error TS1005: ')' expected.
```

**Review by Demonstrandum — APPROVED** [review]

**Pull request merged**